packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/customize-link-in-widget/index.md

```diff
@@ -18,4 +20,18 @@ config['authScheme'] = 'SESSION';
 
 4. Click **Save and Publish**.
 
+### Configure self-service registration in an embedded or self-hosted widget
+
+Before you can configure self-service registration in your embedded or self-hosted widget, make sure [self-service registration](/docs/guides/set-up-self-service-registration/configure-self-service-registration-policy/) is already enabled. Then you can set `feature.registration` in the widget to "true".
```

Comment:
present tense update

Before you can configure self-service registration in your embedded or self-hosted widget, make sure **that you first enable** [self-service registration](/docs/guides/set-up-self-service-registration/configure-self-service-registration-policy/). **You can then set** `feature.registration` in the widget to `true`.